packages/angular/src/lib/abby.service.ts

```diff
@@ -114,13 +123,13 @@ export class AbbyService<
       tap((variant) => (this.selectedVariants[testName as string] = variant)),
       tap((variant) => this.abbyLogger.log(`getVariant(${testName as string}) =>`, variant)),
       map((variant) => {
-        if(lookupObject === undefined) {
+        if (lookupObject === undefined) {
           return variant;
         }
 
         return lookupObject[variant];
       }),
-      shareReplay(1),
+      shareReplay(1)
```

Comment:
guess the project needs some sort of setup for this formatting ping-pong

Reply:
yes, 100%